src/via.ts

```diff
@@ -7,22 +10,59 @@ type ProjectConfig = {
   services: Record<string, ServiceConfig>
 }
 
+async function initConfigDirectory(projectName: string) {
+  await ensureDir(`${viaRoot}/projects`)
```

Comment:
I think we at least have to do:
```suggestion
  if(!projectName) {
    console.log('please specify a project name')
  }
  await ensureDir(`${viaRoot}/projects`)
```
Would be even nicer if we'd ask for a project name if none was given.

Reply:
yeah, would be nice. I didn't wanted to start with interactive prompt at the moment.